packages/suite/src/views/wallet/sign-verify/components/SignAddressInput.tsx

```diff
@@ -51,49 +51,35 @@ const optionToAddress = (option: AddressItem | null) =>
           }
         : null;
 
-export const SignAddressInput = ({
-    name,
-    label,
-    error,
+type SignAddressInputProps = {
+    account?: Account;
+    revealedAddresses: RevealedAddresses;
+} & Omit<SelectProps, 'useKeyPressScroll'>;
+
+export const SignAddressInput: React.FC<SignAddressInputProps> = ({
     account,
     revealedAddresses,
     value,
     onChange,
-    onBlur,
-    isDisabled,
-}: {
-    name: string;
-    label?: string | React.ReactElement;
-    error?: string | React.ReactElement;
-    account?: Account;
-    revealedAddresses: RevealedAddresses;
-    value: string;
-    onChange: (addr: { path: string; address: string } | null) => void;
-    onBlur?: FocusEventHandler;
-    isDisabled?: boolean;
+    ...selectProps
 }) => {
     const { getValue, groupedOptions } = useSignAddressOptions(account, revealedAddresses);
 
     return (
         <Select
-            name={name}
-            label={label}
             value={getValue(value)}
             options={groupedOptions}
             onChange={(addr: AddressItem | null) => onChange(optionToAddress(addr))}
-            onBlur={onBlur}
             noError={false}
-            bottomText={error || null}
-            state={error ? 'error' : undefined}
             isSearchable
-            isDisabled={!!isDisabled}
             placeholder=""
             components={{
                 Option,
                 Input,
                 SingleValue,
             }}
             data-test="@sign-verify/sign-address"
```

Comment:
it looks this is reason why e2e test failing, maybe check if `selectProps` has correct structure because maybe it overrides something